tools/vscode-azurewebpubsub/README.md

```diff
@@ -1 +1,155 @@
+
+   ![Create Resource](./resources/readme/createResource.png)
+
+2. Select **Create Web PubSub Service**. If you are using Socket.IO server, select **Create Web PubSub Service For Socket.IO** instead. See [the document](https://learn.microsoft.com/azure/azure-web-pubsub/socketio-overview) for detail
+
+   ![Create Azure Web PubSub](./resources/readme/createWebPubSub.png)
+    
+3. Type a unique name for the service. Then select resource group, location, pricing tier and unit count for it
+
+## Features
+- View, create, delete, and restart Azure Web PubSub Service
+- View, create, delete hub setting
+- View, create, delete and update event handler
+- View metrics
+- Scale up and scale out
+- Check resource health
+- Regenerate access key
+- Copy connection string or endpoint of the service to clipboard
+- Attach [Azure Web PubSub local tunnel tool](https://www.npmjs.com/package/@azure/web-pubsub-tunnel-tool)
+- View [LiveTrace Tool](https://learn.microsoft.com/azure/azure-web-pubsub/howto-troubleshoot-resource-logs#steps-for-access-key-enabled)
+
+## Create a New Hub Setting
+1. Right click the service where you want to create a new hub setting.
+
+   ![Create Hub Setting](./resources/readme/createHubSetting.png)
+
+2. Click **Create Hub Setting**
+3. Type a name for the hub setting
+4. Select the annoymous connection policy
+4. (Optional) Create a single or multiple event handlers to the hub setting
+
+## Scale 
+
+1. Right click the service you want to scale up or scale out and then click **Advanced**
+
+   ![Advanced](./resources/readme/advanced.png)
+
+2. Click **Scale up** or **Scale Out**
+
+3. Select the new unit count (scale out) or pricing tier (scale up) for your service
+
+   ![Scale Out](./resources/readme/scaleOut.png)
+   ![Scale Up](./resources/readme/scaleUp.png)
+
+## Attach Local Tunnel Tool
+
+1. Expand the hub setting dropdown list, right click the hub setting you want to attach the tunnel tool and then click **Attach Local tunnel**
+
+   ![Attach Local Tunnel](./resources/readme/attachLocalTunnel.png)
+
+2. A tunnel-enabled event handler will be created to support Local Tunnel Tool
+   ![Add Tunnel Handler](./resources/readme/addTunnelEventHandler.png)
+
+   You could customize its system events and user events
+
+   ![View Tunnel Handler](./resources/readme/viewTunnelEventHandler.png)
+
+3. A new terminal will be created and the command to run Local Tunnel Tool on your chosen hub setting will be executed automatically
+   ![View Tunnel Terminal](./resources/readme/viewTunnelTerminal.png)
+
+4. Click **Open Local Tunnel Portal** to see detail in web browser
+
+   ![Local Tunnel Portal](./resources/readme/openTunnelPortal.png)
+
+## Contributing
+
+There are a couple of ways you can contribute to this repo:
+
+- **Ideas, feature requests and bugs**: We are open to all ideas and we want to get rid of bugs! Use the Issues section to either report a new issue, provide your ideas or contribute to existing threads.
```

Comment:
Is this readme included in the vscode extension? If yes, let's also include the issues link

Reply:
updated